napari/utils/colormaps/colormap.py

```diff
@@ -146,23 +151,8 @@ def colorbar(self):
         return make_colorbar(self)
 
 
-class LabelColormap(Colormap):
-    """Colormap that shuffles values before mapping to colors.
-
-    Attributes
-    ----------
-    seed : float
-    use_selection : bool
-    selection : int
-    """
-
-    seed: float = 0.5
-    use_selection: bool = False
-    selection: int = 0
-    interpolation: ColormapInterpolationMode = ColormapInterpolationMode.ZERO
-    background_value: int = 0
-
-    class Config:
+class LabelColormapBase(Colormap):
+    class Config(Colormap.Config):
         # this config is to avoid deepcopy of cached_property
         # see https://github.com/pydantic/pydantic/issues/2763
         # it is required until drop pydantic 1 or pythin 3.11 and older
```

Comment:
Minor grammar/casing issue from previous changes.

```suggestion
        # it is required until we drop Pydantic 1 or Python 3.11 and older
```